app/views/ChargeSummary.scala.html

```diff
@@ -48,38 +42,32 @@
     chargeSummaryNotCodingOutAndNotPayeSelfAssessment: ChargeSummaryNotCodingOutAndNotPayeSelfAssessment,
     govukInsetText : GovukInsetText
 )
-@(currentDate: LocalDate,
-    documentDetailWithDueDate: DocumentDetailWithDueDate, backUrl: String,
-    paymentBreakdown: List[FinancialDetail], paymentAllocations: List[PaymentHistoryAllocations],
-    payments: FinancialDetailsModel, chargeHistoryEnabled: Boolean, paymentAllocationEnabled: Boolean,
-    latePaymentInterestCharge: Boolean, codingOutEnabled: Boolean, isAgent: Boolean = false, btaNavPartial: Option[Html] = None,
-    origin: Option[String] = None, gatewayPage: Option[GatewayPage] = None, isMFADebit: Boolean, documentType: DocumentType, adjustmentHistory: AdjustmentHistoryModel
-    )(implicit request : Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
+@(viewModel: ChargeSummaryViewModel)(implicit request : Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
 
 @import implicitDateFormatter.{longDate, toTaxYearEndDate, toTaxYearStartDate}
 
-@documentDetail = @{documentDetailWithDueDate.documentDetail}
+@documentDetail = @{viewModel.documentDetailWithDueDate.documentDetail}
```

Comment:
another tech debt story raise to refactor this view level logic into view model: MISUV-7944